packages/block-library/src/social-link/edit.js

```diff
@@ -128,19 +130,19 @@ const SocialLinkEdit = ( {
 				<PanelBody
 					title={ sprintf(
 						/* translators: %s: name of the social service. */
-						__( '%s label' ),
+						__( '%s link text' ),
```

Comment:
> As an aside, we could probably make the PanelBody start open, like most of the other blocks. There's no reason to really hide it here.

Thanks @richtabor yes good point, will look into it.
For the labels, I'd lean towards 'link text' as that's how most users call a link text.

Reply:
> I actually think we don't need the social service name here, 

Nice, that would also simplify the label and avoid strings with variable placeholders like `__( '%s label' )`. Those should always be avoided because they're not fully translatable. They are problematic when translators need to swap the order or adjust to genre etc. Example from the current Italian translation:

![Screenshot 2024-03-21 at 14 09 03](https://github.com/WordPress/gutenberg/assets/1682452/1b43e9c0-85c4-4b25-be8d-b73658bce365)

Reply:
> For the labels, I'd lean towards 'link text' as that's how most users call a link text.

I'm not sure I'd characterize this as terminology that most users use, but I'm ok with it.

Reply:
I would like to remind everyone 'Link Text' is what users are familiar with since quite a few years as that's what Classic editor always used:

![Screenshot 2024-03-22 at 13 46 54](https://github.com/WordPress/gutenberg/assets/1682452/a683fad5-be97-46c5-8252-99a06bf4745d)